cpp/src/sender.cpp

```diff
@@ -64,10 +64,17 @@ namespace {
 uint64_t tag_counter = 0;
 }
 
-tracker sender::send(const message &message) {
+tracker sender::send(const message &message,const binary &tag) {
+    pn_delivery_t *dlv;
+    if(!tag.empty())
```

Comment:
this is formatting that clang-format gave you? we'll need to configure the defaults a bit, because in Proton we put braces on the same line, not after `if`

Reply:
Try to create `.clang-format` file containing this

```
Language: Cpp                   # also applies to C
BasedOnStyle: LLVM
IndentWidth:     4
AllowShortIfStatementsOnASingleLine: true
AllowShortLoopsOnASingleLine: true
```

Reply:
I didn't use the clang-format earlier, just did it. Thanks :)